packages/labs/analyzer/src/index.ts

```diff
@@ -5,3 +5,22 @@
  */
 
 export {Analyzer} from './lib/analyzer.js';
+
+export type {
+  Package,
+  Module,
+  Reference,
+  Type,
+  Declaration,
+  VariableDeclaration,
+  ClassDeclaration,
+  LitElementDeclaration,
+  PackageJson,
+  ModuleWithLitElementDeclarations,
+} from './lib/model.js';
+
+export type {AbsolutePath, PackagePath} from './lib/paths.js';
+
+// Any non-type exports below must be safe to use on objects between multiple
+// versions of the analyzer library
+export {getImportsStringForReferences} from './lib/model.js';
```

Comment:
As a follow on, let's consider putting this on the analyzer itself and passing that around. That way we don't have to be careful about this.